language/move-lang/tests/move_check/parser/spec_parsing_inside_fun.exp

```diff
@@ -7,6 +7,16 @@ error[E04003]: built-in operation not supported
    │         Invalid argument to '+'
    │         Found: '()'. But expected: 'u8', 'u64', 'u128'
 
+error[E04007]: incompatible types
+   ┌─ tests/move_check/parser/spec_parsing_inside_fun.move:32:19
+   │
+32 │         spec {} + 1;
+   │         -------   ^
+   │         │         │
+   │         │         Incompatible arguments to '+'
```

Comment:
Is there a way to have this first message point to the binary operator (or the whole expression) as opposed to the RHS of the binop?

Reply:
Didn't see this one, let me fix that before landing